packages/core/src/forms/utils/index.ts

```diff
@@ -5,15 +5,12 @@
  */
 
 import { html } from 'lit-element';
-import { exclamationCircleIcon, checkCircleIcon, ClarityIcons } from '@clr/core/icon';
 import { isVisible } from '@clr/core/internal';
 import { ControlLayout, FormLayout } from './interfaces.js';
 import { CdsInternalControlGroup } from '../control-group/control-group.element.js';
 import { CdsControl } from '../control/control.element.js';
 import { CdsControlMessage } from '../control-message/control-message.element.js';
 
-ClarityIcons.addIcons(exclamationCircleIcon, checkCircleIcon);
-
```

Comment:
I missed one of the side effect issues with the angular cli where it drops the icon register if not in the component.